cv32e40p/bsp/link_gp_relax.ld

```diff
@@ -0,0 +1,311 @@
+/* Script for -z combreloc */
+/* Copyright (C) 2014-2020 Free Software Foundation, Inc.
+   Copyright (C) 2019 ETH Zürich and University of Bologna
+   Copyright (C) 2020 OpenHW Group
```

Comment:
This is a new file and so should have an updated license header.  Ideally it would update the copyright holder and date and use the SPDX Header:
```
# Copyright 2024 Dolphin Design
# SPDX-License-Identifier: Apache-2.0 WITH SHL-2.1
```

Reply:
This is a copy of the original linker script.
I didn't change the header as it was already like that in v1.